app/services/comment_notifications.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class CommentNotifications
```

Comment:
I think this should be renamed `CommentNofificationService` to reflect its purpose.

Reply:
Yeah I think it should be renamed. It is useful for code readers, especially when they see the code at first glance and they will be like "oh okay this is injected in a service".